bc_obps/service/report_service.py

```diff
@@ -41,13 +41,20 @@ def create_report(cls, operation_id: UUID, reporting_year: int) -> Report:
 
         report_version = ReportVersion.objects.create(report=report)
 
+        # Map operation type from full to abbreviated, as expected by report_operation
+        abbreviated_operation_type = operation.type.lower()
+        if abbreviated_operation_type == "linear facility operation":
+            abbreviated_operation_type = "lfo"
+        elif abbreviated_operation_type == "single facility operation":
+            abbreviated_operation_type = "sfo"
+
```

Comment:
The `type` field of the `Operation` model is not constrained, so this code has a pretty high chance of breaking too at some point. I'd remove that constraint on our `ReportOperation` model instead

Reply:
And this way we don't have to test that extra behaviour :p

Reply:
I wasn't sure how critical that constraint on `ReportOperation` was, so erred on the side of changing it. Much more straightforward to just align them unconstrained.